sql/plan.go

```diff
@@ -34,6 +35,15 @@ type planner struct {
 	evalCtx parser.EvalContext
 }
 
+func (p *planner) assignTxn(txn *client.Txn, timestamp time.Time) {
```

Comment:
While this is an assignment, I think this would more typically be named `setTxn`.